src/browser/index.jsx

```diff
@@ -48,14 +44,12 @@ if (!parsedURL.query.index || parsedURL.query.index === null) {
   deeplinkingUrl = remote.getCurrentWindow().deeplinkingUrl;
 }
 
-config.on('update', (configData) => {
-  teams.splice(0, teams.length, ...configData.teams);
```

Comment:
This is mutating the array object which is passed as props to MainPage component.
Mutating objects like this can cause unexpected UI.